source/1_engine/rex_engine/include/rex_engine/core_application.h

```diff
@@ -7,16 +7,16 @@ namespace rex
 {
   struct FrameInfo;
 
-  class IApplication
+  class RexApplication
```

Comment:
Not so sure if I would call this RexApplication ... this does not represent what it actually is IMO. It's a BaseApplication, so maybe just call it that. I know calling it a RexApplication might be cool but it could become a bit confusing in the inheritance tree where this is located

Reply:
ok